### ISSUE TITLE
TypeManager::prepareFieldDefinitions 

### ISSUE THREAD

**Issue by deanfields** [verbb/vizy#160]
**Description**

Attemping to use Graphql that has Vizy (2.0.2) on a very basic install of Craftcms 4.1 and I am receiving this error when simply trying to do ``` { ping }```:

```TypeManager::prepareFieldDefinitions()` has been deprecated. Use `craft\\services\\Gql::prepareFieldDefinition()` instead.```

In my web log it's saying:

```TypeManager::prepareFieldDefinitions()` has been deprecated. Use `craft\\services\\Gql::prepareFieldDefinition()` instead. at /var/www/project/cms/vendor/verbb/vizy/src/gql/interfaces/VizyImageNodeInterface.php:28```

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-4 as 2.0.2",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.3](https://github.com/verbb/vizy/releases/tag/2.0.3)
